examples/tutorials/nuxt.md

```diff
@@ -0,0 +1,423 @@
+---
+title: "Build a Nuxt app with Deno"
+url: /examples/nuxt_tutorial/
+---
+
+[Nuxt](https://nuxt.com/) is a framework that provides an intuitive way to
+create full-stack applications based on [Vue](https://vuejs.org/). It offers
+file-based routing, server-side rendering capabilities, and automatic code
+splitting out of the box. With its modular architecture, Nuxt simplifies the
+development process by providing a structured approach to building Vue
+applications.
+
+In this tutorial, we'll build a simple Nuxt application with Deno that will
+display a list of dinosaurs and allow you to learn more about each one when you
+click on the name:
+
+- [Scaffold a Nuxt app](#scaffold-a-nuxt-app-with-deno)
+- [Setup server API routes](#setup-server-api-routes)
+- [Setup Vue frontend](#setup-vue-frontend)
+- [Add Tailwind](#add-tailwind)
+- [Next steps](#next-steps)
+
+You can find the code for this project in this
+[repo](https://github.com/denoland/examples/tree/main/with-nuxt).
+
+## Scaffold a Nuxt app with Deno
+
+We can create a new Nuxt project using deno like this:
+
+```bash
+deno -A npm:nuxi@latest init
+```
+
+Here, we’re using npm as the package manager. This will create a nuxt-app with
```

Comment:
Is that the best language when talking about the package manager? Might it be more correct to say that we're using Deno as the package manager, and using the package from NPM?

These might be silly semantics, but I'm thinking of landing the point over time, that deno is a package manager too.

So, perhaps...

```suggestion
We'll use Deno to manage our package dependencies, and can grab the Nuxt package from npm. This will create a nuxt-app with
```